### PR TITLE
Metadata updates for release 7.2.7

### PR DESCRIPTION
**Pull request by penmetsaa**
Mar 09, 2016: libphonenumber-7.2.7
Code changes:
 - Update nationalPrefixOptionalWhenFormatting documentation.

Metadata changes:
 - Updated phone metadata for region code(s): CN, IN, MO, TK
 - Updated short number metadata for region code(s): FR
 - Updated carrier data for country calling code(s): 853 (en)
 - Updated / refreshed time zone meta data.

### CODE REVIEW

File: java/release_notes.txt [hunk 1]
```diff
@@ -1,3 +1,10 @@
+Mar 09, 2016: libphonenumber-7.2.7
```
Comment (keghani) [inline java/release_notes.txt]: This should be updated to include the pending changes

Comment (penmetsaa) [inline java/release_notes.txt]: Hi Keghani,

Done with the change.

**Comment by keghani** [comment]
Also update the comment above please :)

**Comment by keghani** [comment]
Thanks Aravind! LG

**Pull request merged**